modules/install/src/main/scala/coursier/install/InstallDir.scala

```diff
@@ -386,22 +386,6 @@ import scala.util.control.NonFatal
 
 object InstallDir {
 
-  private lazy val defaultDir0: Path = {
```

Comment:
Let's keep this logic here, it's fine to have it along `InstallDir` (in case the `install` module is used elsewhere).